server.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const app = express();
+const port = 8080;
+var fs = require('fs');
+//bootup time load data
+var jsonContent = fs.readFileSync('./data.json', 'utf-8');
+var jsonDataPut = null;
+app.get('/', (req, res) => {
+    // json file data fatching
+    var jsonDataGet = jsonContent;
+    console.log(jsonDataGet);
+    res.send(jsonDataGet);
```

Comment:
cant we write this way res.send(jsonContent)

Reply:
Sir, if we use res.send(jsonContent) then after changing in json file we will get updated value without put request.